drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -103,9 +170,19 @@ end
 --- and supports the OnOff cluster. This is done to bypass the
 --- BRIDGED_NODE_DEVICE_TYPE on bridged devices
 local function find_default_endpoint(device, component)
-  local eps = device:get_endpoints(clusters.OnOff.ID)
-  table.sort(eps)
-  for _, v in ipairs(eps) do
+  local switch_eps = device:get_endpoints(clusters.OnOff.ID)
+  local button_eps = device:get_endpoints(clusters.Switch.ID, {feature_bitmap=clusters.Switch.types.Feature.MOMENTARY_SWITCH})
+  local all_eps = {}
```

Comment:
can we call this endpoint_list or something like that? all_eps is vague in my opinion

Reply:
I would prefer to leave this as-is